translations/es.xlf

```diff
@@ -781,6 +768,22 @@
 <trans-unit id="s4cc06ab146dba194">
   <source>Electric transportation</source>
 <target>Transporte eléctrico</target></trans-unit>
+<trans-unit id="sbb254e3763405cb1">
+  <source>By sharing your email, you agree to our</source>
+  <target>Al compartir tu correo electrónico, aceptas nuestros</target>
```

Comment:
This is one risk of machine translation: we've decided in this UI to use the Spanish formal "you", but this is using the informal "you" --- "tu" should be "su", and possibly "aceptas" needs to be conjugated differently?

(I don't actually know enough Spanish to do the translations from scratch myself, but I can recognize issues like this)

Reply:
Oof thank you for catching this.

Reply:
yeah `acepta` should work here

more generally re: formal translations; we recently discussed being more strict with translations (i.e. always do a formal review before merge) whenever possible